### DISCUSSION TITLE
Intellij-community repo cannot be analysed

### DISCUSSION THREAD

**Discussion by guenhter** [github-linguist/linguist#5445] [Classification] [answered]
Hi, when I clone the repo

https://github.com/JetBrains/intellij-community

and run the linguist (as docker container), I don't cat any output.
Can someone confirm this? Is this some settings issue?

**Comment by lildude** [comment, marked as answer]
The issue here is that repo is heeeeaaawg and has a massive object tree and a lot of files so it hitting the guard at:

https://github.com/github/linguist/blob/16c70aef8cd62ca071231a380c69050f5e83c900/lib/linguist/repository.rb#L132-L135

This has been implemented so that repos like this don't negatively impact the analysis of others or excessively hog resources.

**Comment by omearaj** [comment]
Hi @lildude, Uber is evaluating our GHAS product and has hit this issue. They mentioned the languages are not identified in their large repo which does not configure the code scanning matrix in the GitHub action workflow. While they can do it by hand, they did mention the automation aspect. I found this [issue](https://github.com/github/linguist/issues/5594) and read all the other related links. If this is the behavior does it make sense to notify the operator of this limitation via the UI? As it works now, it might appear as a bug to the user. Thanks

**Comment by lildude** [comment]
I've responded in private but for public viewers curious about this: Linguist doesn't really have a say in showing an error/msg or not as it's not responsible for the code that displays the language bar etc... it only produces the data consumed by other parts of GitHub. The maintainers of those sections would need to implement the behaviour to handle this situation.
